src/components/Valve/Child.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+    import { SvelteComponent } from 'svelte';
+    export let icon: typeof SvelteComponent;
+</script>
+
+<button class="rounded-full bg-white p-2" on:click>
+    <svelte:component this={icon} />
```

Comment:
Why wouldn't a `<slot`> have been sufficient here?

Reply:
Would honestly prefer to restrict it with SVGs or images but currently drippy SVGs are stored as svelte components. This is due to the fact that SVGs are valid HTML and hence valid svelte markup which lets us deal with SVGs without the need of an additional Parcel transformer. 
I felt like using a `<slot />` won't dissuade feeding in text or some DOM subtree hence why I opted with this approach. If you happen to know any other way of somewhat restricting the `icon` prop to strictly be an SVG. feel free to suggest.

Reply:
It's honestly alright with me to _not_ dissuade the caller. It just feels a little hacky that we are essentially reimplementing `<slot>`.